### PR TITLE
BXMSDOC-8407-main: Add optional step to set routeHostname variable

### PR DESCRIPTION
**Pull request by csherrar**

### CODE REVIEW

File: doc-content/enterprise-only/openshift/operator-deploy-central-proc.adoc [hunk 1]
```diff
@@ -33,6 +33,17 @@ endif::DM[]
...
 +
+. Optional: To set a custom hostname for the external route, enter a domain in the *Custom hostname to be used on the {CENTRAL} external Route* field, formatted as in the following example:
++
+----
+`businesscentral.example.com`
+----
++
+[NOTE]
+====
+To change the custom hostname, you can modify the `routeHostname` variable.
```
Comment (spolti) [inline doc-content/enterprise-only/openshift/operator-deploy-central-proc.adoc:44]:
```suggestion
To change the custom hostname, you can modify the `routeHostname` property.
```

File: doc-content/enterprise-only/openshift/operator-deploy-kieserver-proc.adoc [hunk 2]
```diff
@@ -17,6 +17,16 @@ Optionally, you can set custom configuration for {KIE_SERVERS}. In this case, de
...
 . Optional: Enter the number of replicas for the {KIE_SERVER} deployment in the *Replicas* field.
+. Optional: To set a custom hostname for the external route, enter a domain in the *Custom hostname to be used on the {KIE_SERVER} external Route* field, formatted as in the following example:
++
+----
+`kieserver.example.com`
+----
++
+[NOTE]
+====
+To change the custom hostname, you can modify the `routeHostname` variable.
```
Comment (spolti) [inline doc-content/enterprise-only/openshift/operator-deploy-kieserver-proc.adoc:28]:
```suggestion
To change the custom hostname, you can modify the `routeHostname` property.
```

File: doc-content/enterprise-only/openshift/operator-deploy-smartrouter-proc.adoc [hunk 3]
```diff
@@ -18,6 +18,17 @@ If you used a custom tag for the image, set the *Image tag* field to this tag.
...
 . Optional: Enter the number of replicas for the Smart Router in the *Replicas* field.
+. Optional: To set a custom hostname for the external route, enter a domain in the *Custom hostname to be used on the Smart Router external Route* field, formatted as in the following example:
++
+----
+`smartrouter.example.com`
+----
++
+[NOTE]
+====
+To change the custom hostname, you can modify the `routeHostname` variable.
```
Comment (spolti) [inline doc-content/enterprise-only/openshift/operator-deploy-smartrouter-proc.adoc:29]:
```suggestion
To change the custom hostname, you can modify the `routeHostname` property.
```

**Review by spolti — CHANGES_REQUESTED** [review]
Hi Clifton, sounds good to me, just a few comments:

- is the release notes update part of this change?
- add a small note instructing customer that the hostname must be valid and resolvable.

File: doc-content/enterprise-only/openshift/operator-deploy-central-proc.adoc (hunk 1 above)
Comment (csherrar) [reply doc-content/enterprise-only/openshift/operator-deploy-central-proc.adoc]: Done

File: doc-content/enterprise-only/openshift/operator-deploy-smartrouter-proc.adoc (hunk 3 above)
Comment (csherrar) [reply doc-content/enterprise-only/openshift/operator-deploy-smartrouter-proc.adoc]: Done

File: doc-content/enterprise-only/openshift/operator-deploy-kieserver-proc.adoc (hunk 2 above)
Comment (csherrar) [reply doc-content/enterprise-only/openshift/operator-deploy-kieserver-proc.adoc]: Done

**Comment by csherrar** [comment]
> Hi Clifton, sounds good to me, just a few comments:
> 
> * is the release notes update part of this change?
> * add a small note instructing customer that the hostname must be valid and resolvable.

Added the note and RN.

File: doc-content/enterprise-only/openshift/operator-deploy-smartrouter-proc.adoc [hunk 4]
```diff
@@ -18,6 +18,19 @@ If you used a custom tag for the image, set the *Image tag* field to this tag.
...
++
+----
+`smartrouter.example.com`
+----
++
+[NOTE]
+====
+The custom hostname must be valid and resolvable.
+
+To change the custom hostname, you can modify the `routeHostname` property.
```
Comment (kaldesai) [inline doc-content/enterprise-only/openshift/operator-deploy-smartrouter-proc.adoc:31]: I can see that you are writing this sentence in the next line. Any specific reason for that? Can we skip the gap? This is for all the notes you have mentioned above. 
```suggestion
[NOTE]
====
The custom hostname must be valid and resolvable. To change the custom hostname, you can modify the `routeHostname` property.
====
```

**Review by kaldesai — APPROVED** [review]
Hey @csherrar, I reviewed all the files. I have a small suggestion otherwise your changes look good to me. 
Thank you so much for this change and for updating the release notes content as well. :)

File: doc-content/enterprise-only/openshift/operator-deploy-smartrouter-proc.adoc (hunk 4 above)
Comment (csherrar) [reply doc-content/enterprise-only/openshift/operator-deploy-smartrouter-proc.adoc:31]: > I can see that you are writing this sentence in the next line. Any specific reason for that? Can we skip the gap? This is for all the notes you have mentioned above.

Reason is that their subject is slightly different, the first is general and about hostname validity while the second is about the property name in the config file. Thoughts?

Comment (csherrar) [reply doc-content/enterprise-only/openshift/operator-deploy-smartrouter-proc.adoc:31]: Thanks, Kalyani and Filippe!

Comment (csherrar) [reply doc-content/enterprise-only/openshift/operator-deploy-smartrouter-proc.adoc:31]: Going to keep them separate for now.

**Pull request merged**